### PR TITLE
OPSEXP-1296 Drop centos 8 support in ansible

### PR DESCRIPTION
**Pull request by himanshu-nigam**
Drop centos 8 support in ansible

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -131,4 +131,4 @@ use_ssl: false
 # Put your domain name here if any (e.g. ecm.alfresco.com). Needs to be empty if unused (don't use dummy values)
 fqdn_alfresco: ""
 
-autogen_unsecure_secrets: false
+autogen_unsecure_secrets: true
```
Comment (alxgomz) [inline group_vars/all.yml:134]: We don't want to allow unsecure setup by defauli

File: Vagrantfile [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-    v.memory = 10240
-    v.cpus = 4
-  end
+config.vm.box = "ubuntu/focal64"
+config.vm.network "private_network", ip: "192.168.56.100"
+config.vm.network "forwarded_port", guest: 80, host: 80
+config.vm.provider "virtualbox" do |v|
+  v.memory = 10240
+  v.cpus = 4
+end
```
Comment (alxgomz) [inline Vagrantfile:8]: Is this indentation change needed?

File: playbooks/secrets-init.yml [hunk 3]
```diff
@@ -47,7 +47,7 @@
           when: vault_init|default('') == 'plugin'
 
         - name: Populate secrets.yml with missing secrets
-          ansible.builtin.shell: "{{ base_folder }}/scripts/generate-secret.sh -s {{ item }} -m {{ gs_mode }} >> {{ secrets_file }}"
+          ansible.builtin.shell: "bash {{ base_folder }}/scripts/generate-secret.sh -s {{ item }} -m {{ gs_mode }} >> {{ secrets_file }}"
```
Comment (alxgomz) [inline playbooks/secrets-init.yml:50]: This has not proven to be any useful. Please revert

File: start [hunk 4]
```diff
@@ -0,0 +1 @@
+vboxdrv
```
Comment (alxgomz) [inline start:1]: Can you explain what this bit is for?

**Review by alxgomz — CHANGES_REQUESTED** [review]
There are also distribution specific files you need to get rid of.

**Review by gionn — CHANGES_REQUESTED** [review]
PR title should contain a summary of the change (like what is currently in the description)

File: playbooks/secrets-init.yml (hunk 3 above)
Comment (himanshu-nigam) [reply playbooks/secrets-init.yml:50]: It does not impact any changes so same work with or without bash

File: start (hunk 4 above)
Comment (himanshu-nigam) [reply start:1]: I guess this is used to start Virtualbox service

File: Vagrantfile (hunk 2 above)
Comment (alxgomz) [reply Vagrantfile:8]: Sorry I don't understand. Why does this change appears in the diff? If you did not changed it on purpose, please revert.

File: playbooks/secrets-init.yml (hunk 3 above)
Comment (alxgomz) [reply playbooks/secrets-init.yml:50]: please be conservative and only comit changes we know had value of fix things

File: start (hunk 4 above)
Comment (alxgomz) [reply start:1]: Again if that is not needed please revert.

**Review by alxgomz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**